Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
```

Comment:
Thank you for all your suggestions.  I added this reference to time.clock_gettime(), time.clock_getres(), and time.clock_settime().  It seems that settime only works for CLOCK_REALTIME though, so maybe it shouldn't link to the whole section?  Also, I did not add any references in time.monotonic(), time.perf_counter(), or time.process_time(), but those seem to be shortcuts to time.clock_gettime()?